.github/workflows/ci.yml

```
@@ -143,6 +151,14 @@
       - name: Install uv
         uses: astral-sh/setup-uv@v3
 
+      - name: Restore cached dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)